rancher2/structure_cluster_v2_rke_config_machine_pool_test.go

```diff
@@ -62,6 +64,12 @@ func init() {
 				"label_one": "one",
 				"label_two": "two",
 			},
+			RKECommonNodeConfig: rkev1.RKECommonNodeConfig{
```

Comment:
Spacing

Reply:
I looked at this, and I couldn't figure out where or how the spacing is wrong. It seems to be consistent with this and other files in this repo?